sound/hda/hdac_stream.c

```diff
@@ -142,6 +142,22 @@ void snd_hdac_stream_stop(struct hdac_stream *azx_dev)
 }
 EXPORT_SYMBOL_GPL(snd_hdac_stream_stop);
 
+/**
+ * snd_hdac_stop_streams - stop all streams if running
+ * @bus: HD-audio core bus
+ */
+void snd_hdac_stop_streams(struct hdac_bus *bus)
```

Comment:
@plbossart do you think snd_hdac_stop_all_streams() is a better name?

Reply:
yes, I wasn't very happy with the name, thanks for the suggestion.

Reply:
fixed